app/src/controllers/project/sagas.js

```diff
@@ -416,14 +416,17 @@ function* watchUpdatePAState() {
 
 function* fetchProject({ payload: { projectKey, fetchInfoOnly } }) {
   try {
-    const project = yield call(fetch, URLS.projectByName(projectKey));
+    let project;
+    if (projectKey) {
+      project = yield call(fetch, URLS.projectByName(projectKey));
+      yield put(setProjectIntegrationsAction(project.integrations));
+    }
     yield put(fetchProjectSuccessAction(project));
```

Comment:
This action can be fired with an undefined `project`.
Does it make sense to wrap it in a condition as well?

Reply:
Done, if there is no project, it will be null.
ProjectSuccess must happen in any case, whether we have a project or not.